L5_deployment_testing.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
```

Comment:
don't commit this file

Reply:
yes, had to leave this in for pending test for binance, will be removed in final commit

Reply:
Binance testing done, fixed.